win32ss/user/user32/lang/de-DE.rc

```diff
@@ -60,9 +60,9 @@ END
 STRINGTABLE
 BEGIN
     IDS_ERROR "Fehler"
-    IDS_NOT_RESPONDING " (Not Responding)"
-    IDS_ASK_TERMINATE "This application is hung up. May I terminate this app?"
-    IDS_HUNG_UP_TITLE "Hung up!"
+    IDS_NOT_RESPONDING " (Reagiert nicht mehr)"
+    IDS_ASK_TERMINATE "Diese Anwendung reagiert nicht mehr. Soll ich sie zum Schließen zwingen?"
```

Comment:
I would change it to: "Diese Anwendung reagiert nicht. Möchten Sie diesen Prozess beenden?"

Reply:
I will also change this